src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -230,13 +230,14 @@ public static string SelectProductNameForDirectory(Platform.XDG_Type dirpath)
         {
             //TODO: XDG_DATA_DIRS implementation as per GitHub issue #1060
 
+            string defaultHomePath = System.Environment.GetEnvironmentVariable("HOME") ?? Directory.GetCurrentDirectory();
```

Comment:
It probably shouldn't default to current working directory if it can't find HOME. What user does the daemon run as? It should still have a HOME defined (one would think).